src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String checkStatus() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Refrain from using magic strings, especially in this context when you're using unicode characters. Using a constant to declare those strings and giving those constants descriptive names will greatly improve readability.